common/src/Model/EntityNode.cpp

```diff
@@ -125,24 +124,21 @@ namespace TrenchBroom {
 
         void EntityNode::doChildWasAdded(Node* /* node */) {
             m_entity.setPointEntity(!hasChildren());
-            nodePhysicalBoundsDidChange(physicalBounds());
+            nodePhysicalBoundsDidChange();
         }
 
         void EntityNode::doChildWasRemoved(Node* /* node */) {
             m_entity.setPointEntity(hasChildren());
-            nodePhysicalBoundsDidChange(physicalBounds());
+            nodePhysicalBoundsDidChange();
         }
 
         void EntityNode::doNodePhysicalBoundsDidChange() {
             invalidateBounds();
         }
 
         void EntityNode::doChildPhysicalBoundsDidChange() {
-            const vm::bbox3 myOldBounds = physicalBounds();
             invalidateBounds();
-            if (physicalBounds() != myOldBounds) {
```

Comment:
These and the other conditions you removed may cause more AABB tree updates, but I suppose that's outweighed by not calling `physicalBound` all the time.

Reply:
Right. That will only affect brush entities, since we don't insert Groups or Layers into the AABB tree.
Actually, it would be better if we didn't insert brush entity bounds into the AABB at all.